lib/aspera/cli/plugins/aoc.rb

```diff
@@ -11,7 +11,7 @@
 module Aspera
   module Cli
     module Plugins
-      class Oncloud < BasicAuthPlugin
+      class Aoc < BasicAuthPlugin
```

Comment:
Thanks, sorry I missed that one... i guess it was the only place where `Oncloud` was used instead of `OnCloud`.

```suggestion
      class AoC < BasicAuthPlugin
```
Should we change this one to match the other `AoC`?